Calculate/ProgramBase.cs

```diff
@@ -0,0 +1,13 @@
+namespace Calculate;
+
+public class ProgramBase
+{
+    public Action<string> WriteLine { get; init; } = s => Console.WriteLine(s);
```

Comment:
```suggestion
    public Action<string> WriteLine { get; init; } = Console.WriteLine;
```
This should work the same way.